controllers/atlasmigration/migrateconf.tmpl

```diff
@@ -0,0 +1,36 @@
+env {
```

Comment:
Move this file to the `templates` directory, respecting the old structure of controllers.